### PR TITLE
STUDIO-6588: adding support for hasHeaders

### PR DESCRIPTION
**Pull request by matiasbargas**

### CODE REVIEW

**Comment by asequeira** [comment]
Commit comment should include the Mule JIRA, please read https://github.com/mulesoft/mule/blob/mule-3.x/CONTRIBUTING.md

File: src/main/java/org/mule/common/metadata/builder/CSVMetaDataBuilder.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import org.mule.common.metadata.ListMetaDataModel;
-import org.mule.common.metadata.StructuredMetaDataModel;
 import org.mule.common.metadata.datatype.DataType;
 
 public interface CSVMetaDataBuilder extends MetaDataBuilder<ListMetaDataModel>
 {
     CSVMetaDataBuilder addField(String fieldName, DataType type);
+
+    CSVMetaDataBuilder setHasHeaders(boolean hasHeaders);
+
```
Comment (asequeira) [inline src/main/java/org/mule/common/metadata/builder/CSVMetaDataBuilder.java]: Remove empty lines.

Comment (marianocapurro) [inline src/main/java/org/mule/common/metadata/builder/CSVMetaDataBuilder.java]: why? I think that empty line improves legibility.

Comment (asequeira) [inline src/main/java/org/mule/common/metadata/builder/CSVMetaDataBuilder.java]: You are right.

**Pull request merged**